### DISCUSSION TITLE
how to set the spawn of a car by coordinates and not a marker on the map

### DISCUSSION THREAD

**Discussion by 6Kegz** [Ppgtjmad/SimpleShops#81] [Q&A] [answered]
how to set the spawn of a car by coordinates and not a marker on the map

**Comment by Ppgtjmad** [comment, marked as answer]
Hello @6Kegz,

### Vehicles shop

Open
`HG/Config/HG_VehiclesShopCfg.h`

Under your shop class, find the **spawnPoints** array : 
```
spawnPoints[] =
{
    {"One set of coordinates",{{x,y,z}}},
    {"Two sets of coordinates",{{x,y,z},{x,y,z}}},
    {"Coordinates + Marker",{{x,y,z},"my_marker_name"}}
    // Etc...
};
```

### Garages

Open
`HG/Config/HG_GaragesCfg.h`

Under your garage class, find the **spawnPoints** array : 
```
spawnPoints[] = 
{
    {"Spawn 1",{{x,y,z}}},
    {"Spawn 2",{{x,y,z},{x,y,z}}}
    // Etc...
};
```

Hope this helps!
